src/sentry/preprod/build_distribution_utils.py

```diff
@@ -17,6 +21,33 @@ def is_installable_artifact(artifact: PreprodArtifact) -> bool:
     return artifact.installable_app_file_id is not None and artifact.build_number is not None
 
 
+PreprodArtifactQuerySet = TypeVar(
+    "PreprodArtifactQuerySet", bound=BaseQuerySet[PreprodArtifact, PreprodArtifact]
+)
+
+
+def annotate_download_count(queryset: PreprodArtifactQuerySet) -> PreprodArtifactQuerySet:
```

Comment:
Wondering if something like the following might be cleaner:

```
class PreprodArtifactQuerySet(BaseQuerySet["PreprodArtifact"]):
    def annotate_download_count(self):
        return self.annotate(
            download_count=Coalesce(
              Sum("installablepreprodartifact__download_count"),
              0,
              output_field=IntegerField(),
            )
        )
```

That way calling it becomes something like `PreprodArtifact.objects.filter(...).annotate_download_count()` which feels a bit nicer if we're mutating the queryset.

Another example of this pattern that I found is `ReleaseQuerySet` if you search for that in the codebase